.github/ISSUE_TEMPLATE/_feature_request.md

```diff
@@ -0,0 +1,20 @@
+
+---
+name: Feature request
```

Comment:
I can imagine we want two special kind of feature requests (what do you think? this is not blocking this PR):

1. a request to add a new `aspect` (like we have `work`, `protein`, etc)
2. a request to add a new panel, where people can post their SPARQL they suggest to be used

Reply:
yes, then should I split this template for both cases?

Reply:
Yes, something like 
name: Feature request, e.g.

* request to add a new `aspect` (in addition to the existing ones like `work`, `protein` etc.)
* request to add a new `panel` within an existing `aspect`, ideally with a suggestion for a suitable SPARQL query